tool/src/main.rs

```diff
@@ -54,12 +58,25 @@ fn main() {
         tas = Tas::new().unwrap();
         println!("TAS created successfully.");
     }
-
-    let script_file = env::args().skip(1).next().unwrap_or("tas.lua".to_string());
-    println!("Executing Script {} ...", script_file);
-    tas.execute(script_file, &config);
+    let script_file = if env::args().len() == 1 {
+        if !std::env::current_dir().unwrap().join("tas.lua").is_file() {
```

Comment:
`Path::new` / `File::open` are looking in the current_dir by default, so no need to manually join it.